Mage.Sets/src/mage/cards/d/DarkApostle.java

```diff
@@ -0,0 +1,158 @@
+package mage.cards.d;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.constants.SubType;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.AbilityWord;
+import mage.constants.CardType;
+import mage.abilities.Ability;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.costs.common.TapSourceCost;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.effects.ContinuousEffectImpl;
+import mage.abilities.keyword.CascadeAbility;
+import mage.constants.Duration;
+import mage.constants.Layer;
+import mage.constants.Outcome;
+import mage.constants.SubLayer;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.stack.Spell;
+import mage.players.Player;
+import mage.game.stack.StackObject;
+import java.util.HashMap;
+import java.util.Map;
+import mage.constants.WatcherScope;
+import mage.watchers.Watcher;
+import mage.util.CardUtil;
+
+
+/**
+ *
+ * @author @stwalsh4118
+ */
+public final class DarkApostle extends CardImpl {
+
+    public DarkApostle(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{3}{R}");
+        
+        this.subtype.add(SubType.ASTARTES);
+        this.subtype.add(SubType.WARLOCK);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(3);
+
+        // Gift of Chaos -- {3}, {T}: The next noncreature spell you cast this turn has cascade.
+        Ability ability = new SimpleActivatedAbility(Zone.BATTLEFIELD,
+            new DarkApostleEffect().setText("The next noncreature spell you cast this turn has cascade."), new ManaCostsImpl("{3}"));
+        ability.addCost(new TapSourceCost());
+        ability.setAbilityWord(AbilityWord.GIFT_OF_CHAOS);
```

Comment:
This doesn't use an established ability word (e.g. Landfall, Threshold, etc), just a flavour word. Replace this with:
`ability.withFlavorWord("Gift of Chaos");`
And undo changes made to AbilityWord.java